src/logic/translations.js

```diff
@@ -120,6 +121,42 @@ const texts = {
       '333mbf': 'Múltiples 3x3x3 a Cegues',
     },
   },
+  es: {
+    eventLabel: 'Categoría',
+    round: 'Ronda',
+    group: 'Grupo',
+    station: 'Estación',
```

Comment:
This is slightly too long to fit. You can enable it in `Configure > General > Print out station number`.